src/main/java/frc/robot/commands/ShiftdownCmd.java

```diff
@@ -0,0 +1,38 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot.commands;
+
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import frc.robot.subsystems.DrivetrainSubsystem;
+
+public class ShiftdownCmd extends CommandBase {
+  /** Creates a new ShiftdownCmd. */
+  private DrivetrainSubsystem drive;
+  public ShiftdownCmd(DrivetrainSubsystem drives) {
+    drive = drives;
```

Comment:
You can use the same variable `drives` if you change this line to `this.drives = drives` where the drives property of this class's instance is set to the supplied drives variable.